storage/innobase/xtrabackup/src/keyring_components.cc

```diff
@@ -0,0 +1,214 @@
+/******************************************************
```

Comment:
can you format this file using clang-format. Oracle also using same standard for formating.